src/TicTacToe.elm

```diff
@@ -11,31 +12,42 @@ main =
 
 -- MODEL
 
-type alias Model = Game
+type PlayState = NotStarted | InProgress Game
+type alias Model = {playState : PlayState}
```

Comment:
That would work, however, my thought was, I'm going to need to extend the record at some point in order to know whether players are human or computer or something like that. 
I'm not sure what the solution is yet.